OneSignalSDK/unittest/src/test/java/com/onesignal/MockOSTime.java

```diff
@@ -0,0 +1,43 @@
+package com.onesignal;
+
+import android.os.SystemClock;
+
+public class MockOSTime implements OSTime {
```

Comment:
Rename to `MockOSTimeImpl` to match our real `OSTimeImpl`?

Reply:
love this one!